react-client/src/graphql/mutations.js

```diff
@@ -75,17 +75,35 @@ export const DROP_COURSE = gql`
 `;
 
 export const ADD_STUDENT = gql`
-  mutation AddStudent($studentNumber: String!, $firstName: String!, $lastName: String!, $email: String!, $phone: String, $password: String!, $favoriteTopic: String, $strongestTechnicalSkill: String) {
-    addStudent(studentNumber: $studentNumber, firstName: $firstName, lastName: $lastName, email: $email, phone: $phone, password: $password, favoriteTopic: $favoriteTopic, strongestTechnicalSkill: $strongestTechnicalSkill) {
+  mutation AddStudent($studentNumber: String!, $firstName: String!, $lastName: String!, $phone: String!, $program: String!, $email: String!, $password: String!, $favoriteTopic: String!, $strongestTechnicalSkill: String!) {
```

Comment:
@dagutierrez-mx If you wanna make this field required you add the `!` on each field, otherwise, just leave it as it is.

Reply:
Not really, I thought it was needed because it was failing to create, but in the end was a schema mismatch